R/stat_stepribbon.R

```diff
@@ -0,0 +1,77 @@
+#' Step ribbon statistic
+#'
+#' Provides stairstep values for ribbon plots
```

Comment:
Can this be more elaborate so the user understands why he wants to use one of the new options rather than the default?

Reply:
added a bit more detail to the header